lib/services/elasticsearch.js

```diff
@@ -542,21 +542,28 @@ module.exports = function (kuzzle, options) {
   };
 
   /**
-   * Reset all indexes
+   * Reset all indexes that the users is allowed to
    *
    * @return {Promise}
    */
   this.deleteIndexes = function (requestObject) {
+    var
+      allowedIndexes = requestObject.data.body.indexes,
+      deletedIndexes = [];
+
     delete requestObject.data.body;
 
     return this.listIndexes(requestObject)
```

Comment:
Since we already use `readEngine.listIndexes` in the `adminController` to get all the indexes that can be deleted, is it necessary to perform another `listIndexes` in the `writeEngine.deleteIndexes` method?